FLT/Mathlib/RepresentationTheory/Basic.lean

```diff
@@ -0,0 +1,40 @@
+import Mathlib.LinearAlgebra.Matrix.GeneralLinearGroup.Defs
+import Mathlib.RepresentationTheory.Basic
+
+open LinearMap
+open scoped TensorProduct
+
+namespace Representation
+
+variable {R V G ι: Type*} [CommRing R] [AddCommMonoid V] [Module R V] [Module.Free R V]
+  [Module.Finite R V] [Group G] [DecidableEq ι] [Fintype ι]
+
+variable (ρ : Representation R G V) (𝓑 : Basis ι R V)
+
+omit [Module.Free R V] [Module.Finite R V] in
+@[simp]
+lemma comp_eq_mul (g h : G): ρ g ∘ₗ ρ h = ρ (g * h) := by
+  simp_all only [map_mul]
+  rfl
+
+noncomputable def gl_map_of_representation_of_basis
```

Comment:
I can't believe that this is what the naming convention says this map is called (but I'm not very good at the naming convention). At the very least you can remove `of_representation` because you're in the Representation namespace right now.